plugins/measure/src/measure.cc

```diff
@@ -130,8 +130,8 @@ void MeasureHandler::render(ViewInterface::Ptr const& vi, cairo_t* cr, Scroom::U
 
   if(selection)
   {
-    Scroom::Utils::Point<double> start(selection->start.x, selection->start.y);
-    Scroom::Utils::Point<double> end(selection->end.x, selection->end.y);
+    Scroom::Utils::Point<double> start(selection->start.x - presentationArea.getLeft(), selection->start.y - presentationArea.getTop());
+    Scroom::Utils::Point<double> end(selection->end.x - presentationArea.getLeft(), selection->end.y - presentationArea.getTop());
```

Comment:
the `Point` class probably needs a constructor that takes a `GdkPoint`.

Prefer doing arithmetic with entire point, instead of individual coordinates. So, eh...
```
auto start = Point(selection->start) - presentationArea.getTopLeft();
```

Reply:
As someone who usually doesn't write c++ I always forget that operator overrides are a thing... Will change this in a bit.

Reply:
Even if operator overrides weren't a thing, I'd probably still recommend you write a function that adds two points, instead of doing it manually. The rest is (unimportant) syntactic sugar